kythe/web/site/examples.md

```diff
@@ -258,6 +258,14 @@ bin: main.cc
 
 ```
 
+```shell
+# cxx_wrapper.sh
+#!/bin/bash -e
+
+$KYTHE_RELEASE_DIR/extractors/cxx_extractor $@
+/usr/bin/c++ $@
```

Comment:
Enquote the "$@"; possibly run the extractor in the background with `&`

Reply:
:+1: